eradicate.py

```diff
@@ -28,176 +28,199 @@
 import io
 import os
 import re
-import sys
 import tokenize
 
 __version__ = '1.0'
 
+class Eradicator:
+    """Eradicate comments."""
+    MULTILINE_ASSIGNMENT_REGEX = re.compile(r'^\s*\w+\s*=.*[(\[{]$')
+    PARTIAL_DICTIONARY_REGEX = re.compile(r'^\s*[\'"]\w+[\'"]\s*:.+[,{]\s*$')
+
+    DEFAULT_WHITELIST = [
```

Comment:
Let's make this immutable. Tuple?

Reply:
It is a `list` because of: `TypeError: can only concatenate list (not "tuple") to list`
here: https://github.com/Cielquan/pydusting/blob/master/eradicate.py#L218

I think its better to have a list, which can be concatenated with another list than need to change types.
If you have a better idea please let me know.

Reply:
I would still prefer a tuple 🙂

Reply:
made it a tuple